### PR TITLE
Refine README.md.

### PR DESCRIPTION
**Pull request by HanFa**
**What this PR does / why we need it**:
Refine README.md for upstream 

* replace all AKO / Avi with the NST Advance Load Balancer

**Which issue(s) this PR fixes**:

Fixes #

**Describe testing done for PR**:

**Special notes for your reviewer**:

**Release note**:

```release-note

```
**New PR Checklist**

- [x] Ensure PR contains only public links or terms
- [x] Use good commit [messages](https://github.com/vmware-tanzu/tanzu-framework/blob/main/CONTRIBUTING.md)
- [x] Squash the commits in this branch before merge to preserve our git history
- [ ] If this PR is just an idea or POC, use a [Draft PR](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests#draft-pull-requests) instead of a full PR
- [x] Add appropriate [labels](https://github.com/vmware-samples/load-balancer-operator-for-kubernetes/labels) according to what type of issue is being addressed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,14 +6,14 @@
 
 - [Quick Start](./docs/quick-start.md)
 
-Load Balancer Operator for Kubernetes is a Cluster API speaking operator for load balancers.
+Load Balancer Operator for Kubernetes is a Cluster API speaking operator for load balancers. It manages the lifecycle of the [Load Balancer and Ingress Service](https://github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/tree/master) resources for Kubernetes clusters.
```
Comment (XudongLiuHarold) [inline README.md:9]: We may want to support other load balancer implementations, we can modify this statement to:
> It manages the lifecycle of load balancers implementations and provides a cluster control plane high availability interface in the multi-cluster scenario.

File: README.md [hunk 2]
```diff
@@ -6,14 +6,14 @@
...
 
 ## Features
 
 1. It reconciles Cluster API objects and provisions Service type LoadBalancer for control plane Machines to achieve HA.
-2. It leverages [Carvel Packaging APIs](https://carvel.dev/kapp-controller/docs/latest/packaging) to lifecycle manage load balancer provider operator. Currently, we support AKO([AVI Kubernetes Operator](https://github.com/vmware/load-balancer-and-ingress-services-for-kubernetes)) as a reference implementation.
+2. It leverages [Carvel Packaging APIs](https://carvel.dev/kapp-controller/docs/latest/packaging) to lifecycle manage load balancer provider operator. Currently, we support [NSX Advance LB Kubernetes Operator](https://github.com/vmware/load-balancer-and-ingress-services-for-kubernetes) as a reference implementation.
```
Comment (XudongLiuHarold) [inline README.md:14]: we **now** support **VMware's** [NSX Advanced LB Kubernetes Operator](https://github.com/vmware/load-balancer-and-ingress-services-for-kubernetes) as a reference implementation.

Comment (XudongLiuHarold) [reply README.md:14]: advanced instead of advance

File: README.md [hunk 3]
```diff
@@ -6,14 +6,14 @@
...
 3. It bridges [Cluster API](https://cluster-api.sigs.k8s.io/) and load balancer provider operator to ensure load balancer resources are cleaned up when cluster is deleted.
-4. For AKO, it also automates the user account creation and injection per cluster.
+4. For the NSX Advance Load Balancer, it also automates the user account creation and injection per cluster.
```
Comment (XudongLiuHarold) [inline README.md:16]: For the NSX Advanced Load Balancer **operator**

File: README.md (hunk 1 above)
Comment (HanFa) [reply README.md]: changed

File: README.md (hunk 2 above)
Comment (HanFa) [reply README.md]: changed

File: README.md (hunk 3 above)
Comment (HanFa) [reply README.md]: fixed

**Review by XudongLiuHarold — APPROVED** [review]
LGTM!

**Pull request merged**